App/localization/legacy_languages/fr.json

```diff
@@ -0,0 +1,87 @@
+{
+	"error_screen_common_sorry": "Désolé !",
```

Comment:
can we remove the legacy ones?

Reply:
I left it because there there are still a few script has not fully migrated yet (at that time). I'll try to fix it this time then remove the legacy.